win/WIN_Window.cpp

```diff
@@ -16,6 +16,17 @@ Window::Window(SDL_Renderer* renderer, gfx::Rectangle const& rect, const char* n
 	// TODO , surface_(surface)
 {
 }
+
```

Comment:
This code can simplify since we are drawing everything and updating everything every frame. 
**SHOULD**

Reply:
I would prefer to keep this the same now, at least until we implement observer patterns to handle the 'changed visuals -> rerender', especially since for the colourpicker (where most of the rerendering occurs) this function is overridden.